app/components/Dashboard.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Put all components related to `Dashboard` into a `components/Dashboard` folder, and put all components related to `jobs` into a `components/Dashboard/jobs` folder.

Let's just remove the `containers` folder, since it's confusing having components in two different folders.